frontend/src/metabase/common/components/SaveQuestionModal/SaveQuestionModal.unit.spec.tsx

```diff
@@ -595,9 +595,11 @@ describe("SaveQuestionModal", () => {
       expect(
         await screen.findByLabelText(/Which tab should this go on/),
       ).toBeInTheDocument();
-      expect(
-        await screen.findByLabelText(/Which tab should this go on/),
-      ).toHaveValue("Foo Tab 1");
+      await waitFor(async () => {
```

Comment:
there are 2 unit test suites that failed due to how tab value are set asynchronously now. So they're modified accordingly.